Add custom BMP icons and NDS banners for games and folders (#64)
LNH-team/pico-launcher: CHANGELOG.md

## [Unreleased]

### Added

> - Custom BMP icons for games and folders, loaded from the SD card

Reviewer: ```suggestion
- Support for custom BMP icons for games and folders - by @tasken
```

Author: ok

> - Support for custom NDS banners (custom titles, subtitles and animated icons) for games and folders

Reviewer: ```suggestion
- Support for custom BMP icons for games and folders - by @tasken
- Support for custom NDS banners (custom titles, subtitles and animated icons) for games and folders - by @tasken
```
I had previously edited my suggestion when I thought about this, but I guess I was too late. Never forget to take credit :p

Author: Thanks

> - Custom NDS banners (custom titles, subtitles, and animated icons) for games and folders, loaded from the SD card

Reviewer: ```suggestion
- Support for custom NDS banners (custom titles, subtitles and animated icons) for games and folders - by @tasken
```

Author: ok
